src/tech/how-we-do-development.md

```diff
@@ -1,13 +1,17 @@
 ---
 title: How we do development
+permalink: /:path/:basename/
+redirect_from:
+  - /guides/how-we-do-development
```

Comment:
Just a note that once this has been merged, I'll either need to rebase and rewrite #971, or close it if this restructuring can resolve the original concern (that it wasn't clear which advice applied to GP and which to D+).

Reply:
I suggest the former after a review.